docs/Deployment/ManualDeploymentExtended.md

```diff
@@ -971,15 +971,13 @@ Full [instructions](https://github.com/lightningnetwork/lnd/blob/master/docs/INS
 
 ```bash
 ~$ sudo apt install make
-~$ wget https://dl.google.com/go/go1.13.linux-amd64.tar.gz
-~$ sha256sum go1.13.linux-amd64.tar.gz
-68a2297eb099d1a76097905a2ce334e3155004ec08cdea85f24527be3c48e856  go1.13.linux-amd64.tar.gz
-~$ sudo tar -C /usr/local -xzf go1.13.linux-amd64.tar.gz
+~$ wget [https://dl.google.com/go/go1.13.linux-amd64.tar.gz](https://go.dev/dl/go1.18.3.linux-amd64.tar.gz) #Note: By default, the go command downloads and authenticates modules using the Go module mirror and Go checksum database run by Google.
```

Comment:
please verify the checksum, there is no way to do this "by default" without doing it explicitly. The checksum is to verify that google didn't changed the binary since the time we checked the sha256sum.

Reply:
ping @LibrtyTools